.github/workflows/ci.yml

```diff
@@ -50,6 +50,7 @@ jobs:
     - run: yarn i18n:checks
     - run: yarn dogfood-lhci
     - run: bash lighthouse-core/scripts/copy-util-commonjs.sh
+    - run: bash lighthouse-core/scripts/copy-axe-valid-langs.sh
```

Comment:
can we skip introducing machinery to update? The list of langs hasn't changed since [March 2018](https://github.com/dequelabs/axe-core/pull/800/). The third-party directory needs a readme anyways, we could just put update instructions in there too